bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -260,22 +260,14 @@ func nameFromAppFile(appFile string) string {
 
 // LoadKfAppCfgFile constructs a KfApp by loading the provided app.yaml file.
 func LoadKfAppCfgFile(cfgfile string) (kftypesv3.KfApp, error) {
-	url, err := netUrl.ParseRequestURI(cfgfile)
-	isRemoteFile := false
-	cwd := ""
+	isRemoteFile, err := utils.IsRemoteFile(cfgfile)
 	if err != nil {
-		return nil, &kfapis.KfError{
-			Code:    int(kfapis.INVALID_ARGUMENT),
-			Message: fmt.Sprintf("Error parsing config file path: %v", err),
-		}
-	} else {
-		if url.Scheme != "" {
-			isRemoteFile = true
-		}
+		return nil, err
 	}
 
 	// If the config file is a remote URI, check to see if the current directory
 	// is empty because we will be generating the KfApp there.
+	cwd := ""
```

Comment:
why initialie to "" as opposed to getCwd()?

Reply:
Is this the current directory or is it a bool indicating whether its empty?

Reply:
nit: Should we clean this code up a bit? isCwdEmpty is returning the current directory but only if its empty.
That's a bit convulated. Should we change isCwdEmpty to isDirEmpty(cwd) and return a bool

Reply:
done